src/azure-cli/azure/cli/command_modules/cosmosdb/_help.py

```diff
@@ -785,6 +785,112 @@
 short-summary: Migrate the throughput of the SQL database between autoscale and manually provisioned.
 """
 
+helps['cosmosdb sql role definition'] = """
+type: group
+short-summary: Manage Azure Cosmos DB SQL role definitions.
+"""
+
+helps['cosmosdb sql role definition create'] = """
+type: command
+short-summary: Create a SQL role definition under an Azure Cosmos DB account.
+examples:
+  - name: Create a SQL role definition under an Azure Cosmos DB account.
+    text: az cosmosdb role definition create -a MyAccount -g MyResourceGroup -b @role-definition.json
+"""
+
+helps['cosmosdb sql role definition delete'] = """
+type: command
+short-summary: Delete a SQL role definition under an Azure Cosmos DB account.
+examples:
+  - name: Create a SQL role definition under an Azure Cosmos DB account.
+    text: az cosmosdb role definition delete -a MyAccount -g MyResourceGroup -i RoleDefinitionId
+"""
+
+helps['cosmosdb sql role definition exists'] = """
+type: command
+short-summary: Checks if an Azure Cosmos DB role definition exists.
```

Comment:
Please remember to use the first person voice.
```suggestion
short-summary: Check if an Azure Cosmos DB role definition exists.
```

Reply:
done